tasks/install_shell.sh

```diff
@@ -431,6 +451,12 @@ do_curl() {
     unable_to_retrieve_package
   fi
 
+  grep "HTTP/2 401" $tmp_stderr 2>&1 >/dev/null
```

Comment:
I don't know that we can safely assume the server replied with HTTP 2. Probably better to just check for:

```suggestion
grep "401 Unauthorized" $tmp_stderr 2>&1 >/dev/null
```